web/yo/app/scripts/services/reportitem.js

```diff
@@ -190,13 +190,18 @@ angular.module('oncokbApp')
                 }else {
                     if(angular.isArray(description)){
                         var str = [];
+                        var allTumorStr = '';
+                        
                         description.forEach(function(e){
                             if(e['Cancer type'].toString().toLowerCase() === 'all tumors' && str.length > 0) {
-                                str.unshift(e.value.toString().trim());
+                                allTumorStr = e.value.toString().trim();
                             }else {
                                 str.push(e.value.toString().trim());
                             }
                         });
+                        if(allTumorStr) {
```

Comment:
should it be if(allTumorStr.length > 0) ? Since empty string will also be true.

Another thing is, should this statement be moved into the description forEach loop? Since currently, allTumorStr always got overwritten by the previous loop.